src/components/MainMenu/SecondLevelMenuItem.js

```diff
@@ -0,0 +1,22 @@
+import {Link} from 'react-router-dom';
```

Comment:
Działa, bo olałeś Chakre :) i zamiast Linka z Chakry używasz z `react-router-dom`. Wszystko ok dla mnie tylko tak zwracam uwagę byś wiedział co się stało ;)
Pod tym linkiem co zapodała @majwro  masz przykład, co trzeba zrobić, by użyć obydwóch w jednym :D 
Konkretnie to do Linka Chakry musisz w propsie `as` zapodać `Linka` z `react-router-dom`, ponieważ jest konflikt Linków, to musisz jeszcze zaimportować jeden z linków i zrobić alias (chociaż nie wiem jak to się zwie w js'ie), tu poniżej zrobiłem alias dla Link'a z reacta `import {Link as RouterLink} from 'react-router-dom';`

```suggestion
import {Link as RouterLink} from 'react-router-dom';
import {Link, MenuItem, MenuList} from "@chakra-ui/react";

function SecondLevelMenuItem({list}) {
    if (Array.isArray(list)) {
        return (
          <MenuList>
            {list.map((element) => (
              <MenuItem key={element.id}>
                <Link as={RouterLink} to={{ pathname: element.url }}>
                  {element.label}
                </Link>
              </MenuItem>
            ))}
          </MenuList>
        );
    }

    return '';
}

```

Reply:
widzę, że tu można z poziomu git'a zmergować, ale wydaje mi się, że to suggestion podmieniłoby pierwszą linię na to co zapodałem, co by było złe ;) to tak na marginesie jakby chciał ktoś to scomitować :D

Reply:
Dzięki @PabloMD za wyjaśnienie. Poprawiłem zgodnie z sugestią.